### PR TITLE
fix: adjusting 'isPLI' flag name

### PR DESCRIPTION
**Pull request by mgaseta**
# Description

Adjusted the flag name on the frontend and swagger documentation. 

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Tested manually.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-330-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-330-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-330-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

**Review by RMCampos — CHANGES_REQUESTED** [review]

File: frontend/src/components/SeedlotRegistrationSteps/OrchardStep/index.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ const OrchardStep = ({
   readOnly
 }: OrchardStepProps) => {
   const queryClient = useQueryClient();
-  const [isPLISpecies] = useState<boolean>(seedlotSpecies.code === 'PLI');
+  const [isPliSpecies] = useState<boolean>(seedlotSpecies.code === 'PLI');
```
[the remark this answers is not in the document]
Comment (mgaseta) [reply frontend/src/components/SeedlotRegistrationSteps/OrchardStep/index.tsx:56]: The species code is in full caps, no need to change here.

[the remark this answers is not in the document]
Comment (craigyu) [reply frontend/src/components/SeedlotRegistrationSteps/OrchardStep/index.tsx:56]: Might be a good idea to keep everything consistent by using `isPliSpecies`

[the remark this answers is not in the document]
Comment (mgaseta) [reply frontend/src/components/SeedlotRegistrationSteps/OrchardStep/index.tsx:56]: @craigyu you mean rename the flag everywhere to `isPliSpecies`?

[the remark this answers is not in the document]
Comment (craigyu) [reply frontend/src/components/SeedlotRegistrationSteps/OrchardStep/index.tsx:56]: I meant the front-end but this is even better!

**Review by RMCampos — APPROVED** [review]
LGTM 🥳

**Pull request merged**